tests/unit/__setup.test.js

```diff
@@ -7,7 +7,7 @@ var Sinon = require('sinon');
 const PORT = 10000;
 
 var core = require('../../src/core');
-core.setNativeBindingsModule(path.resolve('src', 'native-bindings.js'));
+core.setNativeBindingsModule('../src/native-bindings.js');
```

Comment:
I see the `..` bug, but I think for windows using `path.resolve()` is important since it gives the absolute `C:\`-style path.

Reply:
Updated to resolve off __dirname.